account-kit/core/src/actions/createAccount.ts

```diff
@@ -22,32 +22,36 @@ import { getBundlerClient } from "./getBundlerClient.js";
 import { getSigner } from "./getSigner.js";
 import { getSignerStatus } from "./getSignerStatus.js";
 
-type OmitSignerTransportChain<T> = Omit<T, "signer" | "transport" | "chain">;
+//type OmitSignerTransportChain<T> = Omit<T, "signer" | "transport" | "chain">;
```

Comment:
ignore this for now. was just trying to test why type isn't narrowing somewhere.